### PR TITLE
REP-3491 - Tightly Coupled filter wrapper updates

### PR DESCRIPTION
**Pull request by Mario-Lopez**
I updated the Tightly Coupled filter to use the new wrappers.  I removed the unnecessary request wrapping and a couple of odd logging statements (like asking the request for the response content type).  I also removed the "hasBody" check on the response as the new wrapper doesn't support that (and as I understand it the old wrapper barely did anyway).

I also had to set the Content-Length for the whole response to be read by Deproxy.

Also :panda_face: s.

### CODE REVIEW

**Comment by Mario-Lopez** [comment]
Solid 30 test failures.

File: repose-aggregator/components/filters/tightly-coupled-filter/src/main/java/org/openrepose/experimental/filters/servletcontract/TightlyCoupledFilter.java [hunk 1]
```diff
@@ -46,32 +45,24 @@ public void init(FilterConfig filterConfig) throws ServletException {
     public void doFilter(ServletRequest servletRequest, ServletResponse servletResponse, FilterChain filterChain)
             throws IOException, ServletException {
 
-        //Use the repose internal Wrapper to grab a response and modify it
-        MutableHttpServletRequest mutableRequest = MutableHttpServletRequest.wrap((HttpServletRequest) servletRequest);
-        mutableRequest.setInputStream(servletRequest.getInputStream());
-
         //Use a repose internal mutable response
```
Comment (wdschei) [inline repose-aggregator/components/filters/tightly-coupled-filter/src/main/java/org/openrepose/experimental/filters/servletcontract/TightlyCoupledFilter.java]: Great removal of an unused copy/paste of the pattern.

File: repose-aggregator/functional-tests/spock-functional-test/src/test/groovy/features/filters/experimental/tightlyCoupled/TighlyCoupledandOtherFilterTest.groovy [hunk 2]
```diff
@@ -56,30 +54,24 @@ class TighlyCoupledandOtherFilterTest extends ReposeValveTest {
...
+        when:
+        MessageChain mc = deproxy.makeRequest(
+                method        : 'GET',
+                url           : "$reposeEndpoint/get",
+                headers       : ['X-Auth-Token': fakeIdentityService.client_token],
+                defaultHandler: { new Response(200, null, null, body) })
 
         then:
         mc.receivedResponse.code == '200'
+        mc.receivedResponse.body.contains(body)
```
Comment (wdschei) [inline repose-aggregator/functional-tests/spock-functional-test/src/test/groovy/features/filters/experimental/tightlyCoupled/TighlyCoupledandOtherFilterTest.groovy]: Good catch on it being an appends to and not an overwrite of.

**Comment by wdschei** [comment]
Looks good so far. We will wait and see what @jennyvo says.

**Pull request merged**